### PR TITLE
InstructorFeedbackEditCopyUiTest failing on live server #4957

### LINKED ISSUE #4957: InstructorFeedbackEditCopyUiTest failing on live server
java.lang.AssertionError: expected:<The feedback session has been copied. Please modify settings/questions as necessary.> but was:<>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:108)
	at teammates.test.pageobjects.AppPage.verifyStatus(AppPage.java:839)
	at teammates.test.cases.ui.browsertests.InstructorFeedbackEditCopyUiTest.allTests(InstructorFeedbackEditCopyUiTest.java:101)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #4957

### CODE REVIEW

**Comment by kanghj** [comment]
InstructorHomePageUiTest line 418 probably also needs this fix

File: src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditCopyUiTest.java [hunk 1]
```diff
@@ -98,7 +98,7 @@ public void allTests() throws Exception{
         feedbackEditPage.fsCopyToModal.clickSubmitButton();
         feedbackEditPage.waitForPageToLoad();
```
Comment (kanghj) [inline src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditCopyUiTest.java]: We might be able to remove waitForPageToLoad, if using verifyStatusWithRetry is the way to solve this issue. I'm guessing the page load flag in selenium doesn't reset when the page's location is modified using javascript

**Comment by wkurniawan07** [comment]
Strange, that one never failed for me (the other two consistently do in the first run). Anyway, included.

File: src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditCopyUiTest.java (hunk 1 above)
Comment (wkurniawan07) [inline src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditCopyUiTest.java]: You're right, but it's just one of the many things that make #4825 hard.

**Pull request merged**